### ISSUE TITLE
Don't crash when U-label IDN provided on command line

### ISSUE THREAD

**Issue by jsha** [certbot/certbot#3700]
Steps to reproduce:

```
. venv/bin/activate
. tests/integration/_common.sh
certbot_test -d ウェブ.crud.net
```

Expected result: friendly error.

Actual result: Print newline, exit 1. No logs are written.

**Comment by schoen** [comment]
Could be part of #3614, but no matter how we resolve that, we need a better behavior for this situation.

**Comment by sanmai** [comment]
certbot 0.9.3 writes `certbot.log` in cwd wherever it is.

Log file ends with

    ConfigurationError: Internationalized domain names are not presently supported

**Comment by jsha** [comment]
Fixed by #3721.
